src/test/java/org/sagebionetworks/web/unitclient/widget/entity/controller/EntityActionControllerImplTest.java

```diff
@@ -976,6 +978,7 @@ public void testCreateDoiFail() throws Exception {
 	
 	@Test
 	public void testConfigureDoiNotFound() throws Exception {
+		entityBundle.setDoi(null);
 		AsyncMockStubber.callFailureWith(new NotFoundException()).when(mockSynapseClient).getEntityDoi(anyString(), anyLong(), any(AsyncCallback.class));

```

Comment:
With this change, can you completely remove synapseClient.getEntityDoi()?  If so, please do.